# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1253314659 14400
#      Fri Sep 18 18:57:39 2009 -0400
# Node ID 17971b0f18b1af88a929f573783bc0db9512b3c6
# Parent  a307a6f77fb3b70adb333252aac59d13e684cc06
mach-info.cc: don't include oct-types.h

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,12 @@
+2009-09-18  John W. Eaton  <jwe@octave.org>
+
+	* mach-info.cc: Don't include oct-types.h.
+
 2009-09-18  Jaroslav Hajek  <highegg@gmail.com>
 
 	* Array.h (NoAlias): New template class.
 	* dRowVector.cc (linspace): Rewrite.
 	* fRowVector.cc (linspace): Rewrite.
 	* CRowVector.cc (linspace): Rewrite.
 	* fCRowVector.cc (linspace): Rewrite.
 	* dMatrix.cc (linspace): New method.
diff --git a/liboctave/mach-info.cc b/liboctave/mach-info.cc
--- a/liboctave/mach-info.cc
+++ b/liboctave/mach-info.cc
@@ -23,17 +23,16 @@ along with Octave; see the file COPYING.
 
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include "f77-fcn.h"
 #include "lo-error.h"
 #include "mach-info.h"
-#include "oct-types.h"
 
 extern "C"
 {
   double F77_FUNC (d1mach, D1MACH) (const octave_idx_type&);
 }
 
 oct_mach_info *oct_mach_info::instance = 0;
 
